### ISSUE TITLE
[ConCommandSystem] No way to find and/or execute custom commands

### ISSUE THREAD

**Issue by fnky** [baso88/SC_AngelScript#21]
There is no way to find and/or execute custom commands defined in scripts. There should be a `ServerCommand` equivalent for custom commands that does not have to pass through the engine.

**Comment by fnky** [comment]
Code to add this:

```cpp
void CASConCommandSystem_ServerCommand( CASConCommandSystem* pThis, const CString& szCommand )
{
    pThis->Exexcute( CCommand( szCommand.CStr() ), nullptr, true );
}

//In RegisterSCScriptConCommandSystem:
pDocumentation->RegisterObjectMethod( "Executes an Angelscript server command",
    pszObjectName, "void ServerCommand(const string& in szCommand)",
    asFUNCTION( CASConCommandSystem_ServerCommand ), asCALL_CDECL_OBJFIRST );
```

Untested, but should work fine. If not, modify as needed.

**Comment by baso88** [comment]
The fix for this issue will be included in the next update (5.13). Closing.
